joomla/joomla-cms
# Notification email on admin registration approval

Pull Request for Issue #13620 

### Summary of Changes
Sending a notification email, after an Admin activate a user/users, on the user's list of com_users 

### Testing Instructions

- Register as a new user
- Go to the Users list inside the administration, and activate the user/users
- Check the email is sent to the user, notifying him that it has been approved on the site
----------------------------------------------------------------------
- Edit a user that is not activated
- Press the button you will find under the Account Details tab
- Check the user is activated and an email is sent for notifying him that it has been approved on the site
----------------------------------------------------------------------
- Edit a user that is activated, but that has not yet entered in the site for the first time
- Check the same button is there under the Account Details tab, and press it
- Check an email is sent for notifying him that it has been approved on the site

### Expected result
An email is sent to the user, notifying him that it has been approved on the site

### Actual result
The user is activated but is not notified of this change

### Request for feedback (SOLVED)
This change also requires that within the user edit view, a button appears that allows an administrator to activate the user from there. 

For this, I added a new radio field "activation" for handle this under Accounts Details, but this field store a hash when the user It is registered in the site then this new field is not convenient.
![new field - edit view](https://user-images.githubusercontent.com/3956472/39525193-4563a446-4de9-11e8-8939-2d73e64a08c5.png)

I can create a new field under the params fields, and handle this as a param, but I not think this need be under the Basic Settings tab.

So, my doubt is, What is the best way to treat this?

### Documentation Changes Required
- New com_users.user button added. 
- Now when you activate a user on com_users list, send an email
